systemd/systemd.go

```diff
@@ -129,6 +131,36 @@ func Available() error {
 	return err
 }
 
+// Version returns systemd version
+func Version() (int, error) {
```

Comment:
eventually it might be nice to have a Features() function which operates similarly to the apparmor feature system we have in place, but for systemd. Not for this PR, just something to think about

Reply:
that's really useful only if the distros carry very different patch sets or forks, is that the case for systemd?

Reply:
Well in general as we develop more snapd features they may depend on more systemd features that are not available in older versions. TBH, I don't know how useful the set of features exposed through `systemd version` is. If we do end up needing more of this kind of code where systemd version x does this and systemd version y does this other thing, AND there is a flag from `systemd version` output that indicates the presence of the thing we need (and thus the flag is missing in x and present in y), then that's a better check IMO than just hard-coding the fact that systemd version 235 supports feature x and systemd 245 supports feature y.